### DISCUSSION TITLE
The MUX switch button for ROG Zephyrus S17 GX703HSD

### DISCUSSION THREAD

**Discussion by LancerEIX** [seerge/g-helper#1886] [Ideas]
I have a ROG Zephyrus S17 GX703HSD. And this verison of S17 have a MUX switch, and advance optimus.
Every time I want to switch to dGPU only I have to go to bios. (There's also no MUX switch button in Armory Crate).
So is it possible to make a button to switch the dGPU only mode, so that makes switching modes easier.

(Sorry for it is in Chinese) I'm currently using the dGPU only mode, but it still appears “Integrated Graphics+Discrete graphics” in the G-Helper.

<img width="299" alt="Screenshot 2024-01-10 214926" src="https://github.com/seerge/g-helper/assets/156250631/55661690-6549-4418-aba8-7cf123ba6aaa">

**Comment by seerge** [comment]
@LancerEIX do you have a switch in Armoury  ? if not - then it's not possible
